develop/tutorials/articles/110-portlets/07-using-javascript-in-your-portlets/03-using-npm-in-your-portlets/02-configuring-liferay-npm-bundler.markdown

```diff
@@ -91,6 +115,9 @@ override the default output folder.
 asynchronous model, or one by one. The default value is `false`, (parallel), but 
 if you get EMFILE errors, you can disable this.
 
+*verbose:* Sets whether to output detailed information about what the tool is 
+doing.
+
```

Comment:
@izaera is this output in the command prompt, or in a file?

Reply:
verbose is for command line. And yesterday we added a new more: 'dump-report' that writes to a file (see https://github.com/liferay/liferay-npm-build-tools/issues/86). Sorry to notify you in this pull, but we added it late yesterday :-(.

Reply:
No worries. I'll add it now. Thanks!